element-closest.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+/* globals window, Element */
+
+if (window.Element && !Element.prototype.closest) {
+    Element.prototype.closest =
+    function(s) {
```

Comment:
please don't line break after `=` - the `function` part should be on the previous line.